dist/Storable/Storable.pm

```diff
@@ -97,14 +97,7 @@ XSLoader::load('Storable', $Storable::VERSION);
 # Determine whether locking is possible, but only when needed.
 #
 
-sub CAN_FLOCK; my $CAN_FLOCK; sub CAN_FLOCK {
-	return $CAN_FLOCK if defined $CAN_FLOCK;
-	require Config; import Config;
-	return $CAN_FLOCK =
-		$Config{'d_flock'} ||
-		$Config{'d_fcntl_can_lock'} ||
-		$Config{'d_lockf'};
-}
+sub CAN_FLOCK { return 1 }
```

Comment:
Can we assume Flock based on $^O?

Reply:
TL;DR; NO.

Reply:
note that this patch is obsoleted by ad2ec6b54c9968999b7c70984b5a203fa72fd540